server/@types/dto/index.d.ts

```diff
@@ -170,7 +170,7 @@ declare module 'inductionDto' {
   }
 
   export interface InPrisonTrainingInterestDto {
-    trainingType: InPrisonEducationValue
+    trainingType: InPrisonTrainingValue
```

Comment:
Here's a prime example of "Training" and "Education" being used interchangeably. I vote we use the term "training" when it comes to "in-prison" activities.